examples/sentence-transformers-training/sts/readme.md

```diff
@@ -0,0 +1,100 @@
+# Semantic Textual Similarity
+
+Semantic Textual Similarity (STS) assigns a score on the similarity of two texts. In this example, we use the [stsb](https://huggingface.co/datasets/sentence-transformers/stsb) dataset as training data to fine-tune our model. See the following example scripts how to tune SentenceTransformer on STS data:
+
+- **[training_stsbenchmark.py](training_stsbenchmark.py)** - This example shows how to create a SentenceTransformer model from scratch by using a pre-trained transformer model (e.g. [`distilbert-base-uncased`](https://huggingface.co/distilbert/distilbert-base-uncased)) together with a pooling layer.
+- **[training_stsbenchmark_continue_training.py](training_stsbenchmark_continue_training.py)** - This example shows how to continue training on STS data for a previously created & trained SentenceTransformer model (e.g. [`all-mpnet-base-v2`](https://huggingface.co/sentence-transformers/all-mpnet-base-v2)).
+
+## Requirements
+
+First, you should install the requirements:
+
+```bash
+pip install -U sentence-transformers
+pip install git+https://github.com/huggingface/optimum-habana.git
+```
+
+## Single-card Training
+
+To fine tune on the STS task:
+
+1. Choose a pre-trained model `<model_name>` (for example: [bert-base-uncased](https://huggingface.co/google-bert/bert-base-uncased)).
+
+2. Load the training, validation, and test datasets. Here, we use the STS benchmark dataset.
+
+```python
+train_dataset = load_dataset("sentence-transformers/stsb", split="train")
+eval_dataset = load_dataset("sentence-transformers/stsb", split="validation")
+test_dataset = load_dataset("sentence-transformers/stsb", split="test")
+```
+
+3. Execute the script:
+
+	```bash
+		python training_stsbenchmark.py <model_name>
+	```
+
+## Multi-card Training
+
+	For multi-card traning you can use the script of [gaudi_spawn.py](https://github.com/huggingface/optimum-habana/blob/main/examples/gaudi_spawn.py) to execute. There are two options to run the multi-card training by using '--use_deepspeed' or '--use_mpi'. We take the option of '--use_deepspeed' for our example of  multi-card training. 
+
+	```bash
```

Comment:
Same a above readme. Issue with formatting of command.

Reply:
done. thanks